framework/routing/__init__.py

```diff
@@ -30,6 +30,16 @@
     ],
     module_directory='/tmp/mako_modules'
 )
+
+_tpl_lookup_safe = TemplateLookup(
```

Comment:
As far as I can tell the separate objects are needed right now: `TemplateLookup` creates `Template`s directly using its own `default_filters` args, rather than inheriting from the parent.